### PR TITLE
PR for the scripts make_tables.py in order to solve the problem with the energy_offset_array class

### PR DESCRIPTION
**Pull request by JouvinLea**

### CODE REVIEW

File: datasets/hess-crab4/make_tables.py [hunk 1]
```diff
@@ -11,13 +11,16 @@
...
-            name = "hess_events_simulated_" + obs + ".fits"
+        #    name = "hess_events_simulated_" + obs + ".fits"
+            name = "hess_events_simulated_{:06d}.fits".format(obs)
         if filetype == 'background':
-            name = "hess_bkg_offruns_" + obs + ".fits.gz"
+        #    name = "hess_bkg_offruns_" + obs + ".fits.gz"
+            name = "hess_bkg_offruns_{:06d}.fits.gz".format(obs)
         data = dict()
-        data['OBS_ID'] = int(obs[1:])
+        data['OBS_ID'] = obs[1:]
```
Comment (cdeil) [inline datasets/hess-crab4/make_tables.py]: I'm getting this error:
```
In [1]: 1 /0
---------------------------------------------------------------------------
ZeroDivisionError                         Traceback (most recent call last)
<ipython-input-1-5677df635292> in <module>()
----> 1 1 /0

ZeroDivisionError: division by zero
```

File: datasets/hess-crab4/make_tables.py [hunk 2]
```diff
@@ -11,13 +11,16 @@
...
         if filetype == 'events':
-            name = "hess_events_simulated_" + obs + ".fits"
+        #    name = "hess_events_simulated_" + obs + ".fits"
+            name = "hess_events_simulated_{:06d}.fits".format(obs)
         if filetype == 'background':
-            name = "hess_bkg_offruns_" + obs + ".fits.gz"
+        #    name = "hess_bkg_offruns_" + obs + ".fits.gz"
+            name = "hess_bkg_offruns_{:06d}.fits.gz".format(obs)
         data = dict()
-        data['OBS_ID'] = int(obs[1:])
```
Comment (JouvinLea) [inline datasets/hess-crab4/make_tables.py]: I am getting this error:
"""
TypeError: 'int' object has no attribute '__getitem__'
"""

Comment (JouvinLea) [inline datasets/hess-crab4/make_tables.py]: ---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
/Users/jouvin/Desktop/these/test_Gammapy/gammapy-extra/datasets/hess-crab4/make_tables.py in <module>()
     21             name = "hess_bkg_offruns_{:06d}.fits.gz".format(obs)
     22         data = dict()
---> 23         data['OBS_ID'] = obs[1:]
     24         data['TYPE'] = filetype
     25         data['NAME'] = name

TypeError: 'int' object has no attribute '__getitem__'

In [3]: obs[1:]
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-3-183a113df849> in <module>()
----> 1 obs[1:]

TypeError: 'int' object has no attribute '__getitem__'

**Comment by cdeil** [comment]
Looks good to me. Thanks!

**Pull request merged**